From 0b309067e03c2d26da7abbc6aae949e6296810df Mon Sep 17 00:00:00 2001
From: Johannes Bornhold <johannes.bornhold.extern@univention.de>
Date: Fri, 8 Mar 2024 14:58:48 +0100
Subject: [PATCH 2/2] disable license check

The license checking code is not yet supporting the starttls related changes and
it does not yet support a custom port to be used for reaching the ldap server.

See: https://git.knut.univention.de/univention/customers/dataport/upx/iam-issues/-/issues/141
See: https://git.knut.univention.de/univention/ucs/-/merge_requests/1098
---
 .../modules/univention/admin/license.py                         | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/management/univention-directory-manager-modules/modules/univention/admin/license.py b/management/univention-directory-manager-modules/modules/univention/admin/license.py
index 32603eb8fe..f9afae5c07 100644
--- a/management/univention-directory-manager-modules/modules/univention/admin/license.py
+++ b/management/univention-directory-manager-modules/modules/univention/admin/license.py
@@ -190,7 +190,7 @@ class License(object):
         self.__selected = False
 
     def _load_license_via_c_module(self, module):
-        return univention.license.select(module)
+        return 0
 
     def _load_license_via_python(self, module, lo):
         # Try to set the version even if the license load was not successful
-- 
2.39.2

